src/types/Product.types.ts

```diff
@@ -0,0 +1,33 @@
+export interface IProduct {
```

Comment:
This is good. As usual, now let's create an index file that will handle exports from this directory